pydaikin/daikin_brp069.py

```diff
@@ -117,6 +117,26 @@ class DaikinBRP069(Appliance):
         'filter_sign_info': 'filter dirty',
     }
 
+    @staticmethod
+    def parse_response(response_body):
+        """Parse response from Daikin
+
+        Translate swing mode from 2 parameters to 1 (Special case for certain models e.g Alira X)
+        """
+        _LOGGER.debug("Parsing %s", response_body)
+        response = super(DaikinBRP069, DaikinBRP069).parse_response(response_body)
```

Comment:
I'm not super good at working with supclasses and `super` but the [documentation](https://docs.python.org/3/library/functions.html#super) suggests this:

> If the second argument is omitted, the super object returned is unbound. If the second argument is an object, isinstance(obj, type) must be true. If the second argument is a type, issubclass(type2, type) must be true (this is useful for classmethods).

```python
class C(B):
    def method(self, arg):
        super().method(arg)    # This does the same thing as: 
                                               # super(C, self).method(arg)
```

```suggestion
        response = super().parse_response(response_body)
```

Reply:
Oh, `parse_response` is a static method so I guess it is ok to have your code..